app/scripts/directives/unit.coffee

```diff
@@ -22,11 +22,8 @@ angular.module('swarmApp').directive 'unit', ($log, game, commands, options, uti
       #util.utcdoy 1000 * secs
       if isFinite estimate.val
         nonlinear = if not (estimate.unit?.isVelocityConstant?() ? true) then 'less than ' else ''
```

Comment:
Yes. I was wondering if I could not annotate the duration object with the nonlinear value such that the filter could access it. 

The other option would be to expose the nonlinear function and prefix it where required. 




> On 17 Feb 2015, at 8:05 am, Evan Rosson <notifications@github.com> wrote:
> 
> In app/scripts/directives/unit.coffee:
> 
> > @@ -22,11 +22,8 @@ angular.module('swarmApp').directive 'unit', ($log, game, commands, options, uti
> >        #util.utcdoy 1000 * secs
> >        if isFinite estimate.val
> >          nonlinear = if not (estimate.unit?.isVelocityConstant?() ? true) then 'less than ' else ''
> I commented before about prefixing durations with 'less than' if the estimate's not exact (involves units-producing-units). That's still important; I don't want people reporting incorrect times as bugs. (Hopefully I'll improve the estimates soon so it's not necessary.)
> 
> —
> Reply to this email directly or view it on GitHub.
>